examples/ds18b20_broadcaster/ds18b20_broadcaster.c

```diff
@@ -0,0 +1,120 @@
+#include <string.h>
```

Comment:
Nice example! Can you put a header and a copyright notice (ideally public domain/CC0) on it please?